department-of-veterans-affairs/caseflow
# Add props for dropdown and remove logic for hash params

Resolves [APPEALS-37252](https://jira.devops.va.gov/browse/APPEALS-37252)

# Description
Please explain the changes you made here.
- Adds props for the dropdown and feedback url
- Removes hash location logic that was causing issues (since clicking the dropdown adds a hash param)

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. Navigate to the Correspondence intake page
2. Click the dropdown in the top right of the Navigation bar
3. Click on all the different options in there
4. Click on the "Send Feedback" link at the very bottom of the page to the right
5. Expect all those links to work


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
